lib/core/rollup.config.js

```diff
@@ -0,0 +1,298 @@
+import fs from "fs";
+import path from "path";
+import vue from "rollup-plugin-vue";
+import alias from "@rollup/plugin-alias";
+import commonjs from "@rollup/plugin-commonjs";
+import resolve from "@rollup/plugin-node-resolve";
+import replace from "@rollup/plugin-replace";
+import babel from "@rollup/plugin-babel";
+import PostCSS from "rollup-plugin-postcss";
+import simplevars from "postcss-simple-vars";
+import postcssImport from "postcss-import";
+import minimist from "minimist";
+import postcssUrl from "postcss-url";
+import url from "@rollup/plugin-url";
+import nested from "postcss-nested";
+import { terser } from "rollup-plugin-terser";
+import autoprefixer from "autoprefixer";
+import typescript from "rollup-plugin-typescript2";
+import css from "rollup-plugin-css-only";
+
+const postcssConfigList = [
+  postcssImport({
+    resolve(id, basedir) {
+      // resolve alias @css, @import '@css/style.css'
+      // because @css/ has 5 chars
+      if (id.startsWith("@css")) {
+        return path.resolve("./src/assets/styles/css", id.slice(5));
+      }
+
+      // resolve node_modules, @import '~normalize.css/normalize.css'
+      // similar to how css-loader's handling of node_modules
+      if (id.startsWith("~")) {
+        return path.resolve("./node_modules", id.slice(1));
+      }
+
+      // resolve relative path, @import './components/style.css'
+      return path.resolve(basedir, id);
+    }
+  }),
+  simplevars,
+  nested,
```

Comment:
To be honest I'm not a fan of using many plugins for postcss (this is not a change request)